prov/efa/src/rxr/rxr_init.c

```diff
@@ -591,6 +592,12 @@ static int rxr_getinfo(uint32_t version, const char *node,
 			assert(!strcmp(shm_info->fabric_attr->name, "shm"));
 		}
 	}
+
+	goto out;
```

Comment:
This is the success path, can we just do
 ```fi_freeinfo(core_info); return 0``` here 
instead ```goto```?

 Mixing success path and error handling path makes me a little nervous.

Reply:
Good point, fixed.